PayPalNative/src/main/java/com/braintreepayments/api/PayPalNativeFlowCallback.java

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api;
+
+import androidx.fragment.app.FragmentActivity;
+
+/**
+ * Callback for receiving success and error when calling
+ * {@link PayPalNativeClient#tokenizePayPalAccount(FragmentActivity, PayPalRequest, PayPalNativeFlowCallback)}
+ */
+//TODO: i changed the name because i think one call back is enough. The clients wants to know whether it succeeded or failed
+public interface PayPalNativeFlowCallback {
+
+    /**
+     * @param success if the paypal experience is successful.
+     * @param error an exception that occurred while initiating PayPal Native
+     */
+    void onResult(PaymentMethodNonce success, Exception error);
+}
```

Comment:
Changed name from `PayPalNativeStartedFlowCallback` to `PayPalNativeFlowCallback`. My guess is that the client doesn't "care" if the flow fails before the SDK is called or after, it cares only if fails or succeedes. I think this is cleaner, we only have one callback for a successful flow and errors, and thats it. Thoughts?

Reply:
Agreed one callback here makes sense. Maybe `PayPalNativeTokenizeCallback` would make sense as the name?